program/scripts/PDBminer_functions.py

```diff
@@ -193,19 +110,20 @@ def get_structure_df(uniprot_id):
                     uniprot id in terms of pdb files.  
 
     """
-    #find all pdbs for a uniprot id
-    pdb_list = get_pdbs(uniprot_id)
     
-    #if there are no PDBs check for alphafold models
-    if len(pdb_list) == 0:
+    beacons_url = f"https://www.ebi.ac.uk/pdbe/pdbe-kb/3dbeacons/api/uniprot/{uniprot_id}.json?provider=pdbe"
+    response = requests.get(beacons_url)
```

Comment:
I am not really sure that is needed.

For connection error, I think we should implement the 3D-beacon as an API check in the PDBminer file, since the tool should not run at all if there is a connectivity issue (added in commit 2). 

In a case that there is no page for a particular uniprot ID on 3D-beacons the response is "<Response [404]>" and the repsonse.text is {}, this is checked in the next line.